vipps-report-api-faq.md

```diff
@@ -95,6 +94,22 @@ We may add display of the Ledger ID on
 [portal.vipps.no](https://portal.vipps.no)
 in the future.
 
+## How can I get the details for a payment I see in my bank?
+
+The payout details can be found with the `/funds` endpoint, either under date or
+using the feed. Match the settlement id in the bank with `pspReference`.
+
+The Report API is designed to provide updated data independently of the bank payments. There are several reasons for this, including:
+
+1. A merchant can have a negative balance and not get settlement payouts for a long time.
+   Periodic checks of settlements would thus not make sense until the merchant has a positive again.
+1. Merchants have different risk profiles (or could have at least). If we deemed that we would hold
```

Comment:
1. 1. 1... Probably meant to be 1. 2. 3.

Not sure about 2. This is what we have discussed internally but are we ready to tell the world that we could have this in the future? It is a business decision to do this of course. We have made the architecture and APIs ready for such a change but it is not in use today by anyone.

Reply:
Markdown automatically numbers in sequence if we write 1, 1, 1, 1, 1, 1.

Reply:
Item 2: Should be fine with the vagueness that's there now.